src/components/common/PeopleAndRoles/OrgPerson.vue

```diff
@@ -406,6 +406,13 @@ export default class OrgPerson extends Mixins(CommonMixin) {
     return (this.orgPerson?.officer.partyType === PartyTypes.PERSON)
   }
 
+  /** Text same target */
+  get orgTargetType (): string {
```

Comment:
I don't understand this JSDoc comment or getter name. It might make sense to name is similarly to the other getter in this sentence (orgPersonLabel).

It might also be easiest to find this getter if it was close (like, right below) the other one. Kind of a "order of use" ordering.

Reply:
TY